get_fluffy.py

```diff
@@ -1,7 +1,7 @@
 # Fluffy Happiness: Test code to grab pictures of cute animals from the Internet
 # Usage: >> python get_fluffy.py [options]
 # V.A. Moss (vmoss.astro@gmail.com)
-__author__ = "V.A. Moss"
+__author__ = "V.A. Moss smells"
```

Comment:
This is a sub-optimal statement

Reply:
But it is an objectively true statement.

Reply:
Maybe we should qualify what we meet so that it is not vague. I suggest "V.A. Moss smells nice"

Reply:
I find that an acceptable compromise. I accept and ill add that next merge

Reply:
Tally ho, ol' chap.